R/io.R

```diff
@@ -210,19 +210,20 @@ get_ucs_power_plants <- function(ucs_file_path,
                                                     proj4string = CRS(proj4_string)))
 }
 
-#' Get raster value matrix from polygon input areas
+
+#' Get raster frequency per class from polygon input areas
 #'
-#' Get the matrix of raster values represented in the input raster dataset
-#' when restricted to the input watershed polygons for a target city.
+#' Get the frequency per class of raster values represented in the input raster dataset
+#' when restricted to the input watershed polygons for a target city.  This ignores NA.
 #'
 #' @param raster_object character. An object of class RasterLayer.
 #' @param polygon character. A polygon to define spatial boundary of raster value counts (e.g. a given city's watersheds)
-#' @return matrix of raster values in target polygons
+#' @return matrix of raster value and frequency in target polygons
```

Comment:
no longer returns a matrix, right? It's a table of crop types present and their frequency of occurrence.